app/views/users/show.html.erb

```diff
@@ -6,66 +6,19 @@
         <br><br>
         <h3><%= @user.full_name %></h3>
         <h4><%= @user.email %>
-        <!-- <h4>Harvard College</h4>
-        <p><span class="glyphicon glyphicon-user" aria-hidden="true"></span>  Student</p> -->
         <br><br><br>
         <div class="total-points col-centered" data-toggle="tooltip"
              data-placement="bottom" data-trigger="click focus hover"
              title="User score reflects questions authored and votes received on questions">
-          <h2 class="vcenter"><%= @user.karma %></h2>
+          <!--<h2 class="vcenter"><%= @user.karma %></h2>-->
```

Comment:
Delete if unnecessary -- but looks like it breaks user page? Orange circle doesn't have number in it anymore from what I can see.